tests/resources/stargazers.json

```diff
@@ -1,29 +1,23 @@
-{
-  "login": "user",
-  "id": 25349044,
-  "node_id": "MDQ6VXNlcjI1MzQ5MDQ0",
-  "avatar_url": "https://avatars.githubusercontent.com/u/25349044?v=4",
-  "gravatar_id": "",
-  "url": "https://api.github.com/users/user",
-  "html_url": "https://github.com/user",
-  "followers_url": "https://api.github.com/users/user/followers",
-  "following_url": "https://api.github.com/users/user/following{/other_user}",
-  "gists_url": "https://api.github.com/users/user/gists{/gist_id}",
-  "starred_url": "https://api.github.com/users/user/starred{/owner}{/repo}",
-  "subscriptions_url": "https://api.github.com/users/user/subscriptions",
-  "organizations_url": "https://api.github.com/users/user/orgs",
-  "repos_url": "https://api.github.com/users/user/repos",
-  "events_url": "https://api.github.com/users/user/events{/privacy}",
-  "received_events_url": "https://api.github.com/users/user/received_events",
-  "type": "User",
-  "site_admin": true,
-  "blog": "",
-  "hireable": null,
-  "twitter_username": null,
-  "public_repos": 52,
-  "public_gists": 1,
-  "followers": 11,
-  "following": 14,
-  "created_at": "2017-01-25T16:04:54Z",
-  "updated_at": "2022-07-14T22:14:13Z"
-}
\ No newline at end of file
+  {
+      "starred_at": "2016-07-04T08:30:31Z",
+      "user": {
+          "login": "germtb",
```

Comment:
You may want to anonymize this a bit and just replace `germtb` with `user` or something less identifying

Reply:
Thanks, done.